docs/api-mount.rst

```diff
@@ -4,6 +4,12 @@ Mounting UnifyCR
 
 In this section, we describe how to use the UnifyCR API in an application.
 
+.. Note:: **Fortran Compatibility**
+
+   ``unifycr_mount`` and ``unifycr_unmount`` are now usable  with GFortran.
```

Comment:
Thanks @CamStan .  Just because this looks strange to me, let's use the name in all lower case as "gfortran".  That's how I see it most of the time.

Reply:
Well, seems to use lots of variants out on the web:  "GFortran", "Gfortran", "gfortran", so I take that back.  Good how you have it.

Reply:
Oh, we should mention that a Fortran application needs to link with:
```
-lunifycrf -lunifycr_gotcha
```